locations/linked_data_parser.py

```diff
@@ -215,3 +217,14 @@ def parse_enhanced_hotel(ld: dict, item: Feature):
                 item["extras"]["stars"] = stars
             elif isinstance(stars, dict):
                 item["extras"]["stars"] = stars.get("ratingValue")
+
+    @staticmethod
+    def parse_same_as(ld: dict, item: Feature):
+        if same_as := LinkedDataParser.get_clean(ld, "sameAs"):
+            if isinstance(same_as, str):
+                same_as = [same_as]
+            for link in same_as:
+                if "facebook.com" in link:
+                    add_social_media(item, "facebook", link)
```

Comment:
Can we eventually deprecate `Item.facebook` and `Item.twitter` in favour of `add_social_media`?

Reply:
`add_social_media` is a wrapper so if we add a new "social media" eg whatsapp as a top level item, we don't need to edit every spider. It was specifically for that direction, not removing things. Generally I vote for things a reasonable number of spiders output and/or we do "validation" on, being documented. We don't really check things in `extras`.

Tripadvisor may eventually meet my above requirements, in which case we should add it, I'm interested to see just how many POIs we get in the next weekly. I don't envision us deprecating `facebook` or `twitter`.